test/hotspot/jtreg/compiler/c1/TestLinearScanOrderMain.java

```diff
@@ -26,6 +26,7 @@
  * @bug 8207355
  * @compile TestLinearScanOrder.jasm
  * @run main/othervm -Xcomp -XX:+TieredCompilation -XX:TieredStopAtLevel=1
+ *                   -XX:+IgnoreUnrecognizedVMOptions -XX:NMethodSizeLimit=655360
```

Comment:
This test caught one `check_cast<>` issue during development but only on aarch64.
On x64 the test bailed out compilation before that because default `NMethodSizeLimit` was not big enough ((64*K)*wordSize = 524288).